src/__tests__/custom-matchers.test.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+
+import React from 'react';
+import { Button, Text, TextInput, View } from 'react-native';
+import { render } from '..';
+
+describe('custom-matchers', () => {
+  describe('.toBeDisabled', () => {
+    describe('Button', () => {
+      const BUTTON_EXPLICIT_DISABLED = 'button-explicit-disabled';
+      const BUTTON_IMPLICIT_DISABLED = 'button-implicit-disabled';
+      const BUTTON_EXPLICIT_ENABLED = 'button-explicit-enabled';
+      const BUTTON_IMPLICIT_ENABLED = 'button-implicit-enabled';
+      const TestComponent = () => (
+        <View>
+          <Button
+            testID={BUTTON_EXPLICIT_DISABLED}
+            disabled={true}
```

Comment:
This case can be removed, it's the same as below from jsx point of view

Reply:
Removed the 'BUTTON_EXPLICIT_DISABLED' test case